zebra-chain/Cargo.toml

```
@@ -43,6 +43,8 @@ serde-big-array = "0.3.2"
 sha2 = { version = "0.9.4", features=["compress"] }
 thiserror = "1"
 x25519-dalek = { version = "1.1", features = ["serde"] }
+# TODO: change after PR is accepted. This is a commit from the feature/zip-225 branch
+zcash_primitives = { git = "https://github.com/nuttycom/librustzcash", rev = "4b0a3ee3cb8c7472fb225cd6192f0a049d0a38f5" }
```

Since PR https://github.com/zcash/librustzcash/pull/375 is being force-pushed regularly, we need to use a branch here:
```suggestion
zcash_primitives = { git = "https://github.com/nuttycom/librustzcash", branch = "feature/zip-225 " }
```